libvirt/utils_domain_def_test.go

```diff
@@ -12,25 +12,34 @@ func init() {
 }
 
 func TestSplitKernelCmdLine(t *testing.T) {
-	{
-		e := []map[string]string{{"foo": "bar"}, {"foo": "bar", "key": "val"}}
-		r, err := splitKernelCmdLine("foo=bar foo=bar key=val")
-		if !reflect.DeepEqual(r, e) {
-			t.Fatalf("got='%s' expected='%s'", spew.Sdump(r), spew.Sdump(e))
-		}
-		if err != nil {
-			t.Error(err)
-		}
+	e := []map[string]string{{"foo": "bar"}, {"foo": "bar", "key": "val"}}
+	r, err := splitKernelCmdLine("foo=bar foo=bar key=val")
+	if !reflect.DeepEqual(r, e) {
+		t.Fatalf("got='%s' expected='%s'", spew.Sdump(r), spew.Sdump(e))
 	}
+	if err != nil {
+		t.Error(err)
+	}
+}
 
-	{
-		v := "foo=barfoo=bar"
-		r, err := splitKernelCmdLine(v)
-		if r != nil {
-			t.Fatalf("got='%s' expected='%s'", spew.Sdump(r), err)
-		}
-		if err == nil {
-			t.Errorf("Expected error for parsing '%s'", v)
-		}
+func TestSplitKernelInvalidCmdLine(t *testing.T) {
+	v := "foo=barfoo=bar"
```

Comment:
`nitpick` can we use more expressive name then `v`, `e`, `r`?  i know that is testsdata, but still in sake for mantainabilty

Reply:
Sorry, but golang uses `t` for tests. I am following the same convention/thinking.